Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
> When I picked M19 and G5…

I think that's a smart approach because related codes should live together, but lately I'm also checking the RepRap wiki G-code page to see if there's either some code that does what I want, but hasn't been implemented yet, or to see if the code I would like to use already has some well-defined usage.